raven/src/main/java/com/getsentry/raven/DefaultRavenFactory.java

```diff
@@ -75,6 +76,7 @@ public Raven createRavenInstance(Dsn dsn) {
             // https://tomcat.apache.org/tomcat-5.5-doc/servletapi/
             Class.forName("javax.servlet.ServletRequestListener", false, this.getClass().getClassLoader());
             raven.addBuilderHelper(new HttpEventBuilderHelper());
+            raven.addBuilderHelper(new ContextBuilderHelper(raven));
```

Comment:
I don't think this needs to be in this `try`/`catch` block — shouldn't this be unrelated to the HTTP helper?